rand_core/Cargo.toml

```diff
@@ -18,10 +18,11 @@ travis-ci = { repository = "rust-random/rand" }
 appveyor = { repository = "rust-random/rand" }
 
 [features]
-std = ["alloc"]    # use std library; should be default but for above bug
+std = ["alloc", "getrandom", "getrandom/std"]    # use std library; should be default but for above bug
```

Comment:
Why is this coupled to `getrandom`?

Reply:
Do you mean we should support `std` without `getrandom`? We could try, but (1) `getrandom` supports (pretty-much) any `std` platform anyway and (2) I'm not sure Cargo allows us to imply `getrandom/std` only when both `std` and `getrandom` are enabled.